### ISSUE TITLE
ILPP problem

### ISSUE THREAD

**Issue by mitsh** [FirstGearGames/FishNet#558]
**General**
Unity version: 2023.2
Fish-Networking version: 4.0.3-4.0.4
Discord link: https://discord.com/channels/424284635074134018/1034477094731784302/1177660870755627009

**Description**
Everything works fine with Fishnet version 4.0.2. I found the problematic class and simplified to give same error.

**Replication**
```cs
using FishNet.Object;

namespace _Test.CrashTest
{
    public abstract class BaseClass : NetworkBehaviour
    {
        public abstract void InitializeMovementLoop();
    }
}
```

**Expected behavior**
I don't know, it should work as it previous version. May be this usage wrong. I tried to change my logic, like using with public and protected and it didn't work either way.

**Screenshots**
![image](https://github.com/FirstGearGames/FishNet/assets/895798/0c3bc924-89d9-4918-a888-ad60ce95a177)

**Comment by FirstGearGames** [comment]
Okay that should be an easy fix. Have you tried to turn off v3 to v4 helper
yet in the FishNet menu?

**Comment by mitsh** [comment]
yes. I’ve tried to disable the helper. It worked but then in runtime, prediction and movement system stopped working. It gives bunch of sync errors. If you want more information, I can provide it later.

**Comment by FirstGearGames** [comment]
The helper isn't related to any of that. It just runs during codegen to
warn you about any conversation mistakes which may have been made.

**Comment by FirstGearGames** [comment]
I'm still off the the holidays so I'll look into the build issue tomorrow.

**Comment by mitsh** [comment]
okay I’ll wait your redponse. enjoy your holiday! 🥩🍻

**Comment by FirstGearGames** [comment]
Resolved in 4.0.5, will push another small update tomorrow. 

Resolution:
open `SyncTypeProcessor.cs` and goto method `private void ValidateVersion3ToVersion4SyncVars(TypeDefinition td)`
Look for ```cs
                if (methodDef.IsConstructor)
                    continue;
```
and right below it add
```cs
                if (methodDef.IsAbstract)
                    continue;
```
